apps/web/tailwind.config.js

```diff
@@ -2,6 +2,10 @@ const base = require("@calcom/config/tailwind-preset");
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   ...base,
-  content: [...base.content, "../../node_modules/@tremor/**/*.{js,ts,jsx,tsx}"],
+  content: [
+    ...base.content,
+    "../../packages/app-store/routing-forms/**/*.{js,ts,jsx,tsx}",
```

Comment:
Why do we need this?

Reply:
Nothing here was complying JIT tailwind changes only class names that were in the rest of the app